pkg/operator/status.go

```diff
@@ -262,14 +262,27 @@ func (optr *Operator) syncUpgradeableStatus() error {
 		Status: configv1.ConditionTrue,
 		Reason: asExpectedReason,
 	}
+	var updating, degraded bool
 	for _, pool := range pools {
-		degraded := isPoolStatusConditionTrue(pool, mcfgv1.MachineConfigPoolDegraded)
+		// collect updating status but continue to check each pool to see if any pool is degraded
+		if isPoolStatusConditionTrue(pool, mcfgv1.MachineConfigPoolUpdating) {
```

Comment:
Minor: maybe this can be a one-liner like degraded?

(updating = isPoolStatusConditionTrue(pool, mcfgv1.MachineConfigPoolUpdating)

Reply:
I think current code is fine as we want to flag `updating` to `true` when looping over all the pools without overriding previous value.

Reply:
Yep exactly what Sinny said, we're tracking the existence of a true while finishing going thru the pools - we don't want to reset it to false (and lose that value) in the same iteration of the loop.